nxc/helpers/bloodhound.py

```diff
@@ -1,20 +1,36 @@
 #!/usr/bin/env python3
-# -*- coding: utf-8 -*-
+
+from neo4j import GraphDatabase
```

Comment:
for every auth success on nxc, GraphDatabase AuthError, ServiceUnavailable will be imported but not used if bh_enabled = false.
Maybe overkill no ?

Reply:
Hmm yeah we can move it back to the if statement, even though conditional imports aren't great practice.

Reply:
Be careful that it is still in the binary then